### PR TITLE
doc: add an example to explain the diff

### PR DESCRIPTION
**Pull request by AlphaWong**
Hi folks,

Add a more clear example to show the diff between a copy of pod vs ephemeral containers way

Best,
Alpha

### CODE REVIEW

File: content/en/docs/tasks/debug-application-cluster/debug-running-pod.md [hunk 1]
```diff
@@ -256,6 +256,15 @@ If you don't see a command prompt, try pressing enter.
 / #
 ```
 
+You can view a newly created pod `myapp-debug` with using `kubectl get`:
```
Comment (tengqm) [inline content/en/docs/tasks/debug-application-cluster/debug-running-pod.md:259]:
```suggestion
Using `kubectl get pod`, you can see a new pod `myapp-debug` has been created:
```

Comment (reylejano) [reply content/en/docs/tasks/debug-application-cluster/debug-running-pod.md:259]: +1 to @tengqm 's suggestion

**Comment by reylejano** [comment]
@AlphaWong 

I prefer the original.
In this debugging task you use `kubectl debug` and enter the interactive shell of the container 
Naturally you would be debugging in the shell not checking if the new pod is created which is the original flow of this section.

Imo, the updates in this PR breaks the flow of this page.
It adds a `kubectl get pod` between starting an interactive shell with `kubectl debug` and "Now you have an interactive shell that you can use to perform tasks like checking filesystem paths or running the container command manually."

Feel free to reopen this PR if you feel otherwise with `/reopen` 

/close

**Pull request closed without merging**